cep-17.md

```diff
@@ -0,0 +1,527 @@
+# A new recipe format: `jinja` functions in recipes
+
+<table>
+<tr><td> Title </td><td> CEP 17 – A new recipe format (part 3): `jinja` functions in recipes </td>
+<tr><td> Status </td><td> Open </td></tr>
+<tr><td> Author(s) </td><td> Wolf Vollprecht &lt;wolf@prefix.dev&gt;</td></tr>
+<tr><td> Created </td><td> Apr 12, 2024</td></tr>
+<tr><td> Updated </td><td> Jun 10, 2024</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda/ceps/pull/71 </td></tr>
+<tr><td> Implementation </td><td>https://github.com/prefix-dev/rattler-build</td></tr>
+</table>
+
+## Abstract
+
+This CEP is part of the effort to strictly define a new recipe format. The previous CEPs are:
+
+- [CEP 13: A new recipe format](https://github.com/conda/ceps/blob/main/cep-13.md)
+- [CEP 14: A new recipe format: allowed keys and values](https://github.com/conda/ceps/blob/main/cep-14.md)
+
+Historically, `conda-build` recipes have relied on templating with `Jinja` for some
+"dynamic" functionality. For example, many recipes use the `version` of the
+package in multiple places (as package version, in the URL and the tests, for
+example). To make it easy to change recipes, Jinja has been used for some
+light-weight templating.
+
+The "old" recipe format has allowed arbitrary Jinja syntax (including set,
+if/else or for loops). The new recipe format only allows a subset of Jinja with
```

Comment:
```suggestion
The "old" recipe format has allowed arbitrary Jinja syntax (including `set`,
`if`/`else` or `for` loops). The new recipe format only allows a subset of Jinja with
```

Reply:
We should probably also change `old` to `V0` while we're at it.

Reply:
The title of this PR should be changed to reference `V1` recipe format vs `V2`